### PR TITLE
Add more projects from dlang-community

### PR DESCRIPTION
**Pull request by wilzbach**
Sadly not all projects use the same setup and respect $DC.

### CODE REVIEW

File: vars/runPipeline.groovy [hunk 1]
```diff
@@ -344,13 +351,16 @@ DFLAGS=-I%@P%/../imports -L-L%@P%/../libs -L--export-dynamic -L--export-dynamic
...
+        "BlackEdder/ggplotd", // 1m56s
+        "dlang-community/D-Scanner", // 1m40s
+        "rejectedsoftware/diet-ng", // 56s
+        "atilaneves/unit-threaded", //36s
+        "d-gamedev-team/gfm", // 28s
+        "dlang-community/DCD", // 23s
+        "CyberShadow/ae", // 22s
+        "dlang-community/D-YAML", // 15s
+        "dlang-community/libdparse", // 13s
+        "dlang-community/dfmt", // 11s
```
Comment (MartinNowak) [inline vars/runPipeline.groovy]: We should prolly start to write the times to a persisted json file and read this dynamically.
https://jenkins.io/doc/pipeline/steps/workflow-basic-steps/#code-unarchive-code-copy-archived-artifacts-into-the-workspace

File: vars/runPipeline.groovy [hunk 2]
```diff
@@ -234,6 +230,17 @@ def testDownstreamProject (name) {
                     '''
                     break;
 
+                case 'dlang-community/libdparse':
+                    sh 'cd test && DMD=$DC ./run_tests.sh'
```
Comment (MartinNowak) [inline vars/runPipeline.groovy]: We do expose both DMD and DC in the pipeline, just as the install.sh script does.

File: vars/runPipeline.groovy [hunk 3]
```diff
@@ -234,6 +230,17 @@ def testDownstreamProject (name) {
...
+                    break;
+
+                case 'dlang-community/dcd':
+                    sh '''
+                    git submodule update --init --recursive
```
Comment (MartinNowak) [inline vars/runPipeline.groovy]: The travis.sh script does sth. similar, sure just running the travis.yml script fails?

**Comment by wilzbach** [comment]
>  I'll try to run them with their verbatim travis.yml script though.

I wasn't sure whether the `dmd` binary will point to the freshly built one as these two ignore the environment variables.

**Comment by MartinNowak** [comment]
Yes, there is no system-wide dmd on the testers.

**Pull request merged**